config/crd/patches/webhook_in_verticascrutinizes.yaml

```diff
@@ -0,0 +1,8 @@
+# The following patch enables a conversion webhook for the CRD
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: verticascrutinizes.vertica.com
```

Comment:
I don't like the plural form of the CR. Can it be verticascrutinizers instead of verticascrutinizes? I think there are ways to control the plural form when calling `kubebuilder create api`